### PR TITLE
switching gulp use to npm run for cleanliness

### PR DESCRIPTION
**Pull request by grieve**
Since people are good at suggesting and not so much at PR'ing - here ya go

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,9 +13,11 @@
     "Neil McCallion <neil@rehabstudio.com>"
   ],
   "scripts": {
+    "gulp": "gulp",
     "jscs-via-commit": "gulp jscs --via-commit",
     "lint-via-commit": "gulp lint --via-commit",
-    "validate": "gulp jscs --via-commit && gulp lint --via-commit"
+    "validate": "gulp jscs --via-commit && gulp lint --via-commit",
+    "lint": "jshint ."
```
Comment (antwan1986) [inline package.json]: Is this using the `jshint` that `gulp-jshint` pulls down?

Comment (grieve) [inline package.json]: Huh, that's a good question, I didn't add that line.... spooky

Comment (antwan1986) [inline package.json]: Kill it with fire and this PR is good to go!

Comment (grieve) [inline package.json]: But yes anyway, all npm scripts user `node_modules/.bin` by default on PATH

**Pull request merged**